### ISSUE TITLE
Unable to load shaka-player demo or any Widevine DRM protected content on v30.0.0

### ISSUE THREAD

**Issue by eXhumer** [castlabs/electron-releases#182]
Hello. I was recently testing the newly released v30.0.0 and had issues playing Widevine protected content with shaka-player, where player would simply refuse to play any content. The playlist would load without issue, but it seemed like the player wasn't sending any Widevine license request and remaining in paused state. I wanted to check if the shaka-player demo page was working (https://nightly-dot-shaka-player-demo.appspot.com/demo/) and the page seems to not load correctly at all. There is no issue with Widevine playback or the page loading on v29.3.0, but upgrading to v30.0.0 seems to break it. I just wanted to check if there is any issue with the v30.0.0 release, or if the issue is on my end.

My current platform is darwin and arch is arm64. Thanks.

v30.0.0
<img width="1680" alt="Screenshot 2024-04-19 at 7 10 36 AM" src="https://github.com/castlabs/electron-releases/assets/62310242/e314bc36-c854-4c2c-be21-8f18320c9539">

v29.3.0
<img width="1680" alt="Screenshot 2024-04-19 at 7 12 56 AM" src="https://github.com/castlabs/electron-releases/assets/62310242/78c1a3bd-6009-4a4b-801f-982b7f86160f">

**Comment by eXhumer** [comment]
So, I used the BitMovin DRM Stream Test (https://bitmovin.com/demos/drm) and it seems like v30.0.0 doesn't seem to recognize the Widevine CDM loaded, even though it appears to be available based on `components.status` output.

```
{
  oimompecagnajdejgnnjijobebaeigek: {
    status: 'new',
    title: 'Widevine Content Decryption Module',
    version: '4.10.2710.0'
  }
}
```

v29.3.0
<img width="1680" alt="Screenshot 2024-04-19 at 10 36 34 AM" src="https://github.com/castlabs/electron-releases/assets/62310242/a3e17d55-a2c8-431e-bdd0-c2581e736271">

v30.0.0
<img width="1680" alt="Screenshot 2024-04-19 at 10 34 38 AM" src="https://github.com/castlabs/electron-releases/assets/62310242/62b9003a-edea-48a9-b9a5-391dadcc3be9">

**Comment by khwaaj** [comment]
Thanks for the report.

I can confirm this is an issue that appeared between `30.0.0-alpha.6` and `30.0.0-alpha.7` (and it also affects 31.0.0-alpha.1`, albeit a bit differently). I'll be looking into the cause and provide an update once I know more.

**Comment by khwaaj** [comment]
I identified the issue in `v30` so the next release will contain a fix. Turns out that an upstream Chromium change was incorrectly incorporated into Electron and slipped through when we merged it into ECS, which caused key system registration to fail (ultimately leading to EME failures).

I'm confident the issue in `v31` is related, but I'm yet to test a there. There were some additional changes to this key system API in upstream after the change mentioned above, and it seems the Electron team decided to simply remove the key system registration API entirely on the `v31` branch  instead of supporting the API updates, so this is almost certainly the cause of the problem on that branch (and it explains the slight difference in behavior).

**Comment by khwaaj** [comment]
I just published new versions for `v30` and `v31` which fixes the issue:

[v30.0.1+wvcus](https://github.com/castlabs/electron-releases/releases/tag/v30.0.1%2Bwvcus)
[v31.0.0-alpha.2+wvcus](https://github.com/castlabs/electron-releases/releases/tag/v31.0.0-alpha.2%2Bwvcus)

**Comment by eXhumer** [comment]
Just tested the newly released v30.0.1 and I am happy to report that I no longer have any issue with it. Thanks for the quick fix release.
